tests/components/grid.test.ts

```diff
@@ -868,6 +869,18 @@ describe("error tooltip", () => {
     expect(document.querySelector(".o-error-tooltip")).not.toBeNull();
   });
 
+  test("Hovering over a figure should not open popovers", async () => {
+    createFigure(model);
+    await nextTick();
+    setCellContent(model, "C3", "[label](url.com)");
+
+    triggerMouseEvent(".o-figure", "mousemove", 192, 46); // C3 cell is at 192, 46
```

Comment:
If we ever change `DEFAULT_CELL_WIDTH/HEIGHT` this will break. Always use constants when possible.

Reply:
@hokolomopo, I've put into effect the recommended suggestions.